src/View/Helper/ArrayHelper.php

```diff
@@ -25,10 +25,31 @@ class ArrayHelper extends Helper
      * Return array_combine of array using values as keys.
      *
      * @param array $arr The array.
+     *
      * @return array combined array.
      */
     public function combine(array $arr) : array
     {
         return array_combine(array_values($arr), array_values($arr));
     }
+
+    /**
+     * Return array without specified keys.
+     *
+     * @param array $arr  The array.
```

Comment:
I think PHPCS complains if there's more than one whitespace between parameter's name and its comment. Are you using the CakePHP's rules set?

Reply:
Using `PSR2`, but I'm going to fix it